app/models/user.rb

```diff
@@ -1,9 +1,9 @@
 class User < ApplicationRecord
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
-  devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable
+  # Include default devise modules.
+  devise  :database_authenticatable, :registerable, :recoverable, :rememberable,
+          :trackable, :validatable, :confirmable, :omniauthable
```

Comment:
why confirmable and omniauthable?

Reply:
cuando se inicializó devise_token_auth, lo dejé porque en el card no decía que no.